### PR TITLE
Update README to use Vercel

### PR DESCRIPTION
**Pull request by samrobbins85**
Update to use Vercel name.

I used the best image I could find for the bottom link, someone from Vercel might be able to find a better one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+Our mission at Vercel is to make the cloud accessible to everyone. We do that by creating products that improve developer experience, provisioning infrastructure that is globally available, and by teaching the developer community about serverless-related technology. 
 
-Follow us on [Twitter](https://twitter.com/zeithq).
+Follow us on [Twitter](https://twitter.com/vercel).
 
 <br/>
 <br/>
 
-[![](https://assets.zeit.co/image/upload/v1556749970/repositories/vrs/zeit.svg)](https://zeit.co)
+[![](https://assets.vercel.com/image/upload/v1588805858/repositories/vercel/logo.png)](https://vercel.com)
```
Comment (MaxLeiter) [inline README.md:26]: Can you change this to be something like:

```
<p align="center">
  <a href="https://vercel.com">
    <img src="https://assets.vercel.com/image/upload/v1588805858/repositories/vercel/logo.png" height="96">
  </a>
</p>
```

<p align="center">
  <a href="https://vercel.com">
    
  </a>
</p>

**Comment by MaxLeiter** [comment]
Left a small comment. Thanks!

File: README.md (hunk 1 above)
Comment (samrobbins85) [reply README.md]: Of course, pushed a couple commits to fix that

**Comment by MaxLeiter** [comment]
Thanks!

**Pull request merged**